### PR TITLE
Rookout Agent Chart

### PR DESCRIPTION
**Pull request by Zohar2506**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: incubator/rookout/values.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+image:
+  repository: rookout/agent
+  tag: latest
```
Comment (unguiculus) [inline incubator/rookout/values.yaml]: Use stable image

File: incubator/rookout/templates/service.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: {{ template "rookout.fullname" . }}
+  labels:
+    product: rookout
+    app: {{ template "rookout.name" . }}
+spec:
+  selector:
+    app: {{ template "rookout.name" . }}
```
Comment (unguiculus) [inline incubator/rookout/templates/service.yaml]: Ad `release` label to selector.

File: incubator/rookout/templates/service.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: {{ template "rookout.fullname" . }}
+  labels:
```
Comment (unguiculus) [inline incubator/rookout/templates/service.yaml]: Add standard labels.

File: incubator/rookout/templates/deployment.yaml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+      labels:
+        app: {{ template "rookout.name" . }}
+        release: {{ .Release.Name }}
+    spec:
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          env:
+          - name: ROOKOUT_TOKEN
```
Comment (unguiculus) [inline incubator/rookout/templates/deployment.yaml]: Indent one more level for consistency.

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: incubator/rookout/templates/deployment.yaml [hunk 5]
```diff
@@ -0,0 +1,35 @@
+apiVersion: apps/v1beta2
```
Comment (unguiculus) [inline incubator/rookout/templates/deployment.yaml]: Could you use `apps/v1`?

File: incubator/rookout/templates/deployment.yaml [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+          env:
+            - name: ROOKOUT_TOKEN
+              value: {{.Values.token}}
+          ports:
+            - containerPort: 7486
+          livenessProbe:
+            tcpSocket:
+              port: 7486
+          initialDelaySeconds: 15
+          periodSeconds: 20
```
Comment (unguiculus) [inline incubator/rookout/templates/deployment.yaml]: Needs to be indented one more level:
```yaml
initialDelaySeconds: 15
periodSeconds: 20
```

**Review by unguiculus — CHANGES_REQUESTED** [review]
I'd suggest you move the chart to stable directly.

File: incubator/rookout/templates/deployment.yaml (hunk 5 above)
Comment (Zohar2506) [reply incubator/rookout/templates/deployment.yaml]: Done

**Review by unguiculus — CHANGES_REQUESTED** [review]
Now you have two charts, one in stable and one in incubator. Go for the stable one add yourself as maintainer to the `Chart.yaml`.

**Review by Zohar2506 — COMMENTED** [review]
Done

**Comment by Zohar2506** [comment]
Hi @unguiculus is it ok now?

**Comment by unguiculus** [comment]
Looks good but you need to add a readme and NOTES.txt.

File: stable/rookout/README.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+$ helm install --name my-release stable/rookout --set token=YOUR_ORGANIZATIONAL_TOKEN
+```
+
+## Introduction
+
+This chart bootstraps a [Rookout Router](https://docs.rookout.com/docs/installation-agent.html) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- Kubernetes 1.4+ with Beta APIs enabled
```
Comment (unguiculus) [inline stable/rookout/README.md]: 1.9+

File: stable/rookout/templates/deployment.yaml [hunk 8]
```diff
@@ -1,35 +1,40 @@
...
-            periodSeconds: 20
\ No newline at end of file
+    spec:
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          env:
+            - name: ROOKOUT_TOKEN
+              value: {{.Values.token}}
```
Comment (unguiculus) [inline stable/rookout/templates/deployment.yaml]: Add a space after `{{` and before `}}` everywhere.

**Review by unguiculus — APPROVED** [review]

**Comment by Zohar2506** [comment]
Hey @unguiculus can you merge this pull request?
Is there anything missing?

**Comment by Zohar2506** [comment]
@unguiculus

**Pull request merged**